docs/guides/hydrate-app.md

```diff
@@ -97,21 +93,150 @@ the hydrated HTML can be found under the `html` property.
 *Example taken from Ionic Core*
 
 ```javascript
-const results = await hydrate.renderToString(srcHtml, {
-  prettyHtml: true,
-  removeScripts: true
-});
+const results = await hydrate.renderToString(
+  `<my-component first="Stencil" last="'Don't call me a framework' JS"></my-component>`,
+  {
+    fullDocument: false,
+    serializeShadowRoot: true,
+    prettyHtml: true,
+  }
+);
 
 console.log(results.html);
+/**
+ * outputs:
+ * ```html
+ * <my-component class="hydrated sc-my-component-h" first="Stencil" last="'Don't call me a framework' JS" s-id="1">
+ *   <template shadowrootmode="open">
+ *     <style sty-id="sc-my-component">
+ *       .sc-my-component-h{display:block}
+ *     </style>
+ *     <div c-id="1.0.0.0" class="sc-my-component">
+ *       <!--t.1.1.1.0-->
+ *       Hello, World! I'm Stencil 'Don't call me a framework' JS\n" +
+ *     </div>
+ *   </template>
+ *   <!--r.1-->
+ * </my-component>
+ * ```
+ */
 ```
 
 #### renderToString Options
 
-  - `approximateLineWidth` - number
-  - `prettyHtml` - boolean
-  - `removeAttributeQuotes` - boolean
-  - `removeBooleanAttributeQuotes` - boolean
-  - `removeEmptyAttributes` - boolean
-  - `removeHtmlComments` - boolean
-  - `afterHydrate` - boolean
-  - `beforeHydrate` - boolean
+##### `approximateLineWidth`
+
+__Type:__ `number`
+
+Determines when line breaks are being set when serializing the component.
+
+##### `prettyHtml`<br />
```

Comment:
Nit-picky, but mind dropping the `<br/>` tokens in this file? Think it's a little easier to read that way and keeps the formatting consistent

Reply:
👍 done